flask_jwt_extended/utils.py

```diff
@@ -132,7 +133,8 @@ def _get_jwt_manager():
                            "application before using this method")
 
 
-def create_access_token(identity, fresh=False, expires_delta=None, user_claims=None):
+def create_access_token(identity, fresh=False, expires_delta=None, user_claims=None,
+                        headers=None):
```

Comment:
I'm curious if you think it would provide value to add a custom decorator to add the headers to created tokens, along the lines of how `@jwt.user_claims_loader` works? (https://flask-jwt-extended.readthedocs.io/en/stable/add_custom_data_claims.html). Or would that be too much for this feature?